components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -7,26 +7,117 @@ documented [here](https://mozilla.github.io/glean_parser/metrics-yaml.html).
 
 When adding a new metric, the workflow is:
 * Decide on which metric type you want to use.
-* Add a new entry to metrics.yaml.
-* Add code to your project to record into the metric by calling Glean.
+* Add a new entry to `metrics.yaml`.
+* Add code to your project to record into the metric by calling glean.
 
 **Important**: as stated [here](../../README.md#before-using-the-library), any new data collection requires
 documentation and data-review. This is also required for any new metric automatically collected
 by glean.
 
+# The `metrics.yaml` file
```

Comment:
I feel this whole section is too long at this place and this doc is not snappy anymore.
I may be off here, so feel free to counter.

Could we...
- Move this further down?
   - after the "choosing a metric type"? My assumption being that looking for a metric type to match their problem is what people come here first with, then they want details on how that works in detail.
   - Or instead have one short section on "getting kickstarted with the metrics.yaml file" through one hands-on example... and put the other details here further down or in a separate file?
- Add a top-level TOC around here or further up?

Reply:
What I was really trying to address here is that these docs didn't cover any of the required metric parameters (they just leave them out with `...`) which makes it hard to just dive in.  Maybe the thing to do is to have a single complete example near the top here and move the details about each parameter to the bottom?

Reply:
I've fleshed out the main example, and moved these details to the bottom.